README.md

```diff
@@ -190,7 +190,21 @@ import localeData from "moment/locale/zh-cn.js";
 import fp from "lodash/fp.js";
 ```
 
-_Note how unlike some Node.js usages, we include the ending `.js` here. File extensions are required in browsers; unlike in Node, [we do not have the luxury](#the-nodejs-module-resolution-algorithm) of trying multiple file extensions until we find a good match. Fortunately, including file extensions also works in Node.js; that is, if everyone uses file extensions for submodules, their code will work in both environments._
+#### Extension-less imports
+
+It is also common in the Node.js ecosystem to import files without including the extension. [we do not have the luxury](#the-nodejs-module-resolution-algorithm) of trying multiple file extensions until we find a good match. Fortunately, an import map can also include these cases.
+
+```diff
+ {
+   "imports": {
+     "lodash": "/node_modules/lodash-es/lodash.js",
+     "lodash/": "/node_modules/lodash-es/",
++    "lodash/fp": "/node_modules/lodash-es/fp.js",
```

Comment:
This is an example of deep package imports, not really "extension-less imports".

In the context of the node modules WG discussions, extensionless imports refer to relative specifiers as well.

Reply:
it's kinda both, but i can see the confusion. do you think there is a better example?